ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -116,8 +115,11 @@ export const AdminInstitutionEdit = withUrlParams()(class extends React.Componen
       });
 
     // TODO(RW-6933): Implement new institution admin UI with CT support.
-    this.setState(fp.set(['institution', 'tierEmailAddresses'],
```

Comment:
To confirm, it seems like the bug was that this still referenced `'tierEmailAddresses'`, which is a field that you removed in the later version of the PR (and `tierDomains` below)

Reply:
Right. I found textare's `onBlur` not working, then saw `onBlur` call this method, then found this issue